### ISSUE TITLE
Doc: Runtime patching on Node.js

### ISSUE THREAD

**Issue by d9k** [D-Pow/MockRequests#14]
Does this library supports runtime patching (without the use of webpack at all)?

inside `node` interpreter:

```
MR = require('mock-requests');
fetch = require('node-fetch');

MR.setMockUrlResponse('http://t/1', {
  mockExample: 1
});

fetch('http://t/1').then((response) => response.json()).then(data => console.log(data));
// request fails
```

**Comment by d9k** [comment]
```
fetch = require('node-fetch');
global.fetch = fetch
MR = require('mock-requests');

MR.setMockUrlResponse('http://localhost:1111/t', {
  mockExample: 1
});

fetch('http://localhost:1111/t').then((response) => response.json()).then(data => console.log(data));
// request fails
```

**Comment by d9k** [comment]
OK, found a solution

```
fetch = require('node-fetch');
global.fetch = fetch
MR = require('mock-requests');
fetch = global.fetch

MR.setMockUrlResponse('http://localhost:1111/t', {
  mockExample: 1
});

fetch('http://localhost:1111/t').then((response) => response.json()).then(data => console.log(data));
// request fails
```

**Comment by D-Pow** [comment]
This is a great catch, @d9k!

As you mentioned, [node-fetch recommends its users to patch the `global.fetch` function](https://github.com/node-fetch/node-fetch#loading-and-configuring-the-module) in order to use it. That's why I've typically favored using [isomorphic-fetch](https://www.npmjs.com/package/isomorphic-fetch) in the past for both browser and node polyfills -- because it automatically patches everything for the user.

That being said, this should be made specifically clear in the MockRequests docs so that other users aren't left scratching their heads over why it's not working as expected. I'll leave this issue open until I push the update.

**Comment by D-Pow** [comment]
This should be fixed in [PR #17](https://github.com/D-Pow/MockRequests/pull/17). Please let me know if you still encounter any issues!
